hellomama_registration/utils.py

```diff
@@ -231,3 +243,10 @@ def get_available_metrics():
                 "registrations.source.%s.sum" % source.user.username)
 
     return available_metrics
+
+
+def normalise_string(string):
+    """ Strips trailing whitespace from string, lowercases it and replaces
+        spaces with underscores
+    """
+    return ((string.strip()).lower()).replace(" ", "_")
```

Comment:
We probably want this to be something like `re.sub(r'\W+', '_', string)` to be absolutely sure (in case there are any strange characters)